scripts/compile_parsers.makefile

```diff
@@ -0,0 +1,16 @@
+#
+# compile_parsers.makefile
+# Stephan Seitz, 2021-09-09 21:36
+#
+CC?=cc
+
+all: parser.so
+
+parser.o: src/parser.c
+	$(CC) -c src/parser.c -std=c99 -fPIC -I./src
+
+scanner.o: src/scanner.cc
+	$(CC) -c src/scanner.cc -std=c++17 -fPIC -I./src
+
+parser.so: parser.o scanner.o
+	$(CC) parser.o scanner.o -o parser.so -shared -Os -lstdc++
```

Comment:
Does stdc++ exist on Mac? Or will clang just do some compat magic?

Reply:
Tried to use `-lstdc++` on my M1 and it works with Apple clang 12.0.5

Reply:
Yes, clang just tries to behave like GCC